inc/namespace.php

```diff
@@ -336,7 +336,7 @@ function fix_rest_attachment_urls( WP_REST_Response $response, WP_Post $attachme
 
 	$data = $response->get_data();
 
-	if ( ! empty( $data['media_details']['sizes'] ) ) {
+	if ( ! empty( $data['media_details'] ) && is_array( $data['media_details'] ) && ! empty( $data['media_details']['sizes'] ) && is_array( $data['media_details']['sizes'] ) ) {
```

Comment:
As we're using an `empty()` check here this can be reduced down to the following:
```suggestion
	if ( ! empty( $data['media_details']['sizes'] ) && is_array( $data['media_details']['sizes'] ) ) {
```

Reply:
Hey @roborourke, This is still causing the following error while trying to access stdClass object of `$data['media_details']` as an array.

> PHP Fatal error:  Uncaught Error: Cannot use object of type stdClass as array

Reply:
Hm, sorry. Clearly not with it today. I’ll patch it shortly

Reply:
No problem, I've created https://github.com/humanmade/asset-manager-framework/pull/44 :)